server/routes/api/v1/petTypesRouter.js

```diff
@@ -27,5 +27,18 @@ petTypesRouter.get("/:type", async (req, res) => {
   }
 })
 
+petTypesRouter.get("/:type/:id", async (req, res) => {
+  try {
+    const pet = await Pet.findPetById(req.params.id)
```

Comment:
If this endpoint is only dealing with the Pet resource, it should live in the pets router, not the _pet types router_